README.md

```diff
@@ -116,6 +116,19 @@ To create a CSV file for the benchmark result :
 
     $ ./bench_name | sed '2d;s/ \{1,\}//g' > bench_name.csv
 
+Building secp256k1 - Using vcpkg
+------------
+
+You can download and install secp256k1 using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
```

Comment:
"Building" is not "download and install" :)

This file contains no references to any package manager which provides the `secp256k1` package. Seeing no reasons to make an exception for the vcpkg.